lib/solveEquation/EquationOperations.js

```diff
@@ -242,4 +240,12 @@ function performTermOperationOnExpression(expression, op, term) {
   return newNode;
 }
 
+function containsSymbolNode (node, symbolName) {
+  if (Node.Type.isSymbol(node)) { return node.name === symbolName; }
+  else if (Node.Type.isParenthesis(node)) { return (containsSymbolNode(node.content, symbolName)); }
+  else if (node.args) { return !!node.args.find((child) => containsSymbolNode(child, symbolName)); }
+  else { return false; }
+}
```

Comment:
see comment above. As we're used to, this is merely a simple recursive tree traversal looking for the symbol. It's a lot simpler and more reliable, imo, than `Symbols.getLastSymbolTerm(`